### PR TITLE
docs: add local predict endpoint test readme

### PR DESCRIPTION
**Pull request by jaredscheib**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -176,3 +176,11 @@ The SavedModel will be saved in a timestamped directory under `/tmp/mnist_saved_
...
 
 Use [`saved_model_cli`](https://www.tensorflow.org/guide/saved_model#cli_to_inspect_and_execute_savedmodel) to inspect and execute the SavedModel.
+
+## Getting predictions using a script
+1. Clone down this repository
+1. Create a virtual environment (i.e. using `virtualenv`) in the root of the cloned directory
+1. Run `pip install -r requirements.txt`
+1. Run `python serving_rest_client_test.py --url https://staging-services.paperspace.io/model-serving/de1b8j937fgmkx:predict`
+
+_Note: you may encounter an error about installing Python as a framework related to *matplotlib*. If so, try updating the backend of your matploglib using [the recommendations here](https://stackoverflow.com/questions/21784641/installation-issue-with-matplotlib-python). If that doesn't work, try installing _python3_ with Homebrew and possibly use the [uninstall and reinstall method here](https://github.com/scikit-optimize/scikit-optimize/issues/637).
```
Comment (mkulaczkowski) [inline README.md]: Matplotlib is now optional not required. It will simply say that displaying the image wont be possible but otherwise its fully capable/

Comment (jaredscheib) [reply README.md]: hm, yeah i saw that message but unfortunately even after performing the above commands, i still ran into the issue. i haven't been able to successfully run it on my machine

Comment (mkulaczkowski) [reply README.md]: @jaredscheib can you be more specific

Comment (mkulaczkowski) [reply README.md]: @jaredscheib i am avaliable today if you could show me that

**Comment by jaredscheib** [comment]
@mkulaczkowski I think we can close this for now since the new docs with `python3` and `requirements-local.txt` fixed this on my machine. If other users start to experience problems, we can revisit this PR.

**Pull request closed without merging**